scripts/exregional_run_fcst.sh

```diff
@@ -450,6 +450,24 @@ cat > itag <<EOF
 EOF
 fi
 
+#
+#-----------------------------------------------------------------------
+#
+# Choose namelist file to use
+#
+#-----------------------------------------------------------------------
+#
+STOCH="FALSE"
+if [ "${DO_ENSEMBLE}" = "TRUE" ] && ([ "${DO_SPP}" = "TRUE" ] || [ "${DO_SPPT}" = "TRUE" ] || [ "${DO_SHUM}" = "TRUE" ] || \
+   [ "${DO_SKEB}" = "TRUE" ] || [ "${DO_LSM_SPP}" =  "TRUE" ]); then
+     STOCH="TRUE"
+fi
+if [ "${STOCH}" == "TRUE" ]; then
+  ln_vrfy -sf ${FV3_NML_STOCH_FP} ${DATA}/${FV3_NML_FN}
+ else
+  ln_vrfy -sf ${FV3_NML_FP} ${DATA}/${FV3_NML_FN}
+fi
+
```

Comment:
This is needed to pick the right namelist file for stochastic physics. We currently directly modify the input namelist directly in SRW. This was needed to get `community_ensemble_2memss_stoch` working @JeffBeck-NOAA